README.md

```diff
@@ -286,7 +286,10 @@ If true, a field will be marked as required on PUT and POST pages.
 If true, a field will be displayed, but not editable. It's data will still be added to the PUT request.
 
 ###### ``useInUrl`` (boolean)
-If true, a field can be used as a paramter in a PUT url. Otherwise only fields retreived in the original GET can be used as paramters. It's data will still be added to the PUT request body.
+If true, a field can be used as a paramter in a PUT url. Otherwise only fields retreived in the original GET can be used as parameters. By default it's data will still be added to the PUT request body (unless ``omitFromPayload`` is true).
+
+###### ``omitFromPayload`` (boolean)
```

Comment:
Could you also add a "omitFromPayploadIfEmpty"?

If the field is empty it won't be sent. But if it's full - it will.

Reply:
See:
https://github.com/dsternlicht/RESTool/issues/69

Reply:
Also, is this applied to POST requests as well?

And maybe it should omitted from payload by default?

Reply:
Ok I see what #69 needs. I'll have a think and get back to you.

Reply:
Ok. So for now I’ll merge these changes, and you could add this later.